HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 History
 -------
 
+0.5.1 (2022-03-10)
+-----------------
+* Fixed issues and pending PR
```

Comment:
I think, it would be great to have a list of most important changes in the changelog:
```suggestion
* Updated connector's urlencoding logic for proper array encoding #287;
* Updated InfobloxObject's fetch method to raise `InfobloxFetchGotMultipleObjects` exception #288;
* Fix a bug when calling abstracted class from_dict with V4 & V6 subclass #282;
* Objects pass through the paging argument #305.
```